pxr/base/arch/fileSystem.cpp

```diff
@@ -1111,76 +1111,27 @@ int ArchFileAccess(const char* path, int mode)
 
 // https://msdn.microsoft.com/en-us/library/windows/hardware/ff552012.aspx
 
-#define MAX_REPARSE_DATA_SIZE  (16 * 1024)
-
-typedef struct _REPARSE_DATA_BUFFER {
-    ULONG   ReparseTag;
-    USHORT  ReparseDataLength;
-    USHORT  Reserved;
-    union {
-        struct {
-            USHORT  SubstituteNameOffset;
-            USHORT  SubstituteNameLength;
-            USHORT  PrintNameOffset;
-            USHORT  PrintNameLength;
-            ULONG   Flags;
-            WCHAR   PathBuffer[1];
-        } SymbolicLinkReparseBuffer;
-        struct {
-            USHORT  SubstituteNameOffset;
-            USHORT  SubstituteNameLength;
-            USHORT  PrintNameOffset;
-            USHORT  PrintNameLength;
-            WCHAR   PathBuffer[1];
-        } MountPointReparseBuffer;
-        struct {
-            UCHAR  DataBuffer[1];
-        } GenericReparseBuffer;
-    };
-} REPARSE_DATA_BUFFER, *PREPARSE_DATA_BUFFER;
-
 std::string ArchReadLink(const char* path)
 {
-    HANDLE handle = ::CreateFile(path, GENERIC_READ, 0, NULL, OPEN_EXISTING,
-        FILE_FLAG_OPEN_REPARSE_POINT |
-        FILE_FLAG_BACKUP_SEMANTICS, NULL);
+	bool success = false;
```

Comment:
This indent is a tab character, but should be 4 spaces to match the indentation in the rest of the file.  Please convert all tabs to 4 spaces and push again.